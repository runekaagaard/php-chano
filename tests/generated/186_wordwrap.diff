001- this is a short
001+ Fatal error: Call to undefined method DtlIter::wordwrap() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/186_wordwrap.php on line 5
002- paragraph of
003- text.
004-   But this line
005- should be
006- indented