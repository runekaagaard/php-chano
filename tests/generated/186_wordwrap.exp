this is a short
paragraph of
text.
  But this line
should be
indented