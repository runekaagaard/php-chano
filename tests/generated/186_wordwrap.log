
---- EXPECTED OUTPUT
this is a short
paragraph of
text.
  But this line
should be
indented
---- ACTUAL OUTPUT
Fatal error: Call to undefined method DtlIter::wordwrap() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/186_wordwrap.php on line 5
---- FAILED
