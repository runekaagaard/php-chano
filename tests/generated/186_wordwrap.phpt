--TEST--
A generated testfile for the "wordwrap" filter.
--FILE--
<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => 'this is a short paragraph of text.
  But this line should be indented')));
foreach ($items as $i) echo $i->input->wordwrap(15);
--EXPECT--
this is a short
paragraph of
text.
  But this line
should be
indented