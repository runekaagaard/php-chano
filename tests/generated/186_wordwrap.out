Fatal error: Call to undefined method DtlIter::wordwrap() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/186_wordwrap.php on line 5